.github/CODEOWNERS

```
@@ -1,2 +1,5 @@
 # Every PR gets a review from an internal Expensify engineer
 * @Expensify/pullerbear
+
+# Assign the Design team to review changes to our styles
+src/styles/ @Expensify/Design @Expensify/pullerbear
```

```suggestion
src/styles/ @Expensify/design @Expensify/pullerbear
```

Sorry for leading you astray in Slack, I think this should actually be little `d` and you had it correct the first time.